pkgs/development/python-modules/psycopg/default.nix

```diff
@@ -50,14 +50,6 @@ let
       libpq = "${postgresql.lib}/lib/libpq${stdenv.hostPlatform.extensions.sharedLibrary}";
       libc = "${stdenv.cc.libc}/lib/libc.so.6";
     })
-
-    (fetchpatch {
```

Comment:
If I still could, I would just have pushed nit fixup to remove the unused input 🙃

Reply:
Fixed in 1327fd5accf1ae5ebee90a17023077fea352a8f5